examples/header-group.html

```diff
@@ -0,0 +1,80 @@
+<template id=header-group>
+
+<style>
+
+  header-group
+    { display: block }
+
+  header-group
+    hgroup
+      { background: #bada55; color: var(--darkest-color) }
+
+</style>
+
+<hgroup>
+  <h1><a href=/examples/index.html>snuggsi ツ</a></h1>
+  <slot name=header><h2>try an example</h2></slot>
+</hgroup>
+
+</template>
+
+<script>
+const tag = `header-group`
+
+Element `header-group`
+
+(class extends HTMLElement {
+
+  static onimportload (event) {
```

Comment:
yeah sounds like events should go with events. i'm ok with shipping an early version that uses an improper name if it'll take some time to figure it out. not a lot of folks depending on this yet